src/core/echo.js

```
@@ -1,12 +1,93 @@
+        case 'r': out += '\r'; break;
+        case 't': out += '\t'; break;
+        case 'v': out += '\v'; break;
+        case '\\': out += '\\'; break;
+        case '0': {
+          let oct = '';
+          let j = i + 1;
+          while (j < s.length && oct.length < 3 && /[0-7]/.test(s[j])) {
+            oct += s[j++];
+          }
+          if (oct) {
+            out += String.fromCharCode(parseInt(oct, 8));
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)